docs/admin/management.rst

```diff
@@ -525,22 +523,22 @@ importuserdata
 
 .. django-admin:: importuserdata <file.json>
 
-Imports userdata from file created by :djadmin:`dumpuserdata`
+Imports user data from a file created by :djadmin:`dumpuserdata`
 
 importusers
 -----------
 
 .. django-admin:: importusers --check <file.json>
 
-Imports users from JSON dump of Django auth_users database.
+Imports users from JSON dump of the Django auth_users database.
 
 
 .. django-admin-option:: --check
 
-    With this option it will just check whether given file can be imported and
-    report possible conflicts on usernames or e-mails.
+    With this option it will just check whether a given file can be imported and
```

Comment:
What is "It", Weblate? What are the conflicts? How is that resolved? A note could be added.